src/main/java/com/hydratereminder/HydrateReminderPersonalityType.java

```diff
@@ -9,6 +9,7 @@
 {
     SIMPLE("Simple"),
     FUN("Fun");
```

Comment:
Looks like something here is causing the build to fail.

I believe swapping the semicolon after FUN for a comma should fix the issue.

Reply:
Yes, I missed that one! Fixed now.